examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,179 @@
+"""
+
+import os
+import re
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import nncf
+import numpy as np
+import openvino.runtime as ov
+import torch
+from sklearn.metrics import accuracy_score
+from openvino.offline_transformations import compress_quantize_weights_transformation
+from torchvision import datasets
+from torchvision import models
+from torchvision import transforms
+from tqdm import tqdm
+
+FOOD101_CLASSES = 101
+ROOT = Path(__file__).parent.resolve()
+DATASET_PATH = Path(sys.argv[1]) if len(sys.argv) > 1 else ROOT / 'dataset'
+CHECKPOINT_URL = 'https://huggingface.co/AlexKoff88/mobilenet_v2_food101/resolve/main/pytorch_model.bin'
+
+
+def fix_names(state_dict):
+    state_dict = {key.replace('module.', ''): value for (key, value) in state_dict.items()}
+    return state_dict
+
+
+def load_checkpoint(model):  
+    checkpoint = torch.hub.load_state_dict_from_url(CHECKPOINT_URL, progress=False)
+    weights = fix_names(checkpoint['state_dict'])
+    model.load_state_dict(weights)
+    return model
+
+
+def validate(model, val_loader):
+    predictions = []
+    references = []
+
+    output_name = model.outputs[0]
+
+    for images, target in tqdm(val_loader):
+        pred = model(images)[output_name]
+        predictions.append(np.argmax(pred, axis=1))
+        references.append(target)
+
+    predictions = np.concatenate(predictions, axis=0)
+    references = np.concatenate(references, axis=0)  
+    return accuracy_score(predictions, references)
+
+
+def ov_convert(model, args):
+    onnx_model_path = f'{tempfile.gettempdir()}/model.onnx'
+    torch.onnx.export(model, args, onnx_model_path, verbose=False)
+
+    ov_model = ov.Core().read_model(onnx_model_path)
+    compress_quantize_weights_transformation(ov_model)
+    return ov_model
+
+
+def ov_benchmark(model, verbose=True):
+    ov_model_path = f'{tempfile.gettempdir()}/model.xml'
+    ov.serialize(model, ov_model_path)
+
+    command = f'benchmark_app -m {ov_model_path} -d CPU -api async -t 15'
+    cmd_output = subprocess.check_output(command, shell=True)
+    if verbose:
+        print(*str(cmd_output).split('\\n')[-9:-1], sep='\n')
+    match = re.search(r"Throughput\: (.+?) FPS", str(cmd_output))
+    return float(match.group(1))
+
+
+def ov_model_size(ir_path, m_type='Mb'):
+    model_size = os.path.getsize(ir_path) + os.path.getsize(ir_path.replace('xml', 'bin'))
+    for t in ['bytes', 'Kb', 'Mb']:
+        if m_type == t:
+            return model_size
+        model_size /= 1024
+    return model_size
+
+###########################################################################
+# Create a PyTorch model and dataset
+###########################################################################
+normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
+                                    std=[0.229, 0.224, 0.225])
+val_dataset = datasets.Food101(
+    root=DATASET_PATH,
+    split = 'test', 
+    transform = transforms.Compose([
```

Comment:
It would be great to add a link to a description of why we use these particular transformations. It looks like `ImageNet` transformations, isn't it?